build.sh

```diff
@@ -5,7 +5,7 @@ set -e
 workdir=$(dirname $(realpath $0))
 version=$(cat version 2>/dev/null)
 folder_name="polaris-server-release_${version}"
-if [ -n $GOOS ] && [ -n $GOARCH ]
+if [ -n "$GOOS" ] && [ -n "$GOARCH" ]
```

Comment:
Instead of this check, I think this is better:

```shell
GOOS=`go env GOOS`
GOARCH=`go env GOARCH`
```

WDYT ?

Reply:
BTW, chmod +x on this file.

Reply:
you are right